lib/labels/cloud.go

```diff
@@ -160,7 +158,7 @@ func (l *CloudImporter) Sync(ctx context.Context) error {
 	m := make(map[string]string)
 	for key, value := range tags {
 		if !types.IsValidLabelKey(key) {
-			l.Log.Debugf("Skipping cloud tag %q, not a valid label key.", key)
+			l.Log.DebugContext(ctx, "skipping cloud tag due to invalid label key", "tag", key)
```

Comment:
```suggestion
			l.Log.DebugContext(ctx, "Skipping cloud tag due to invalid label key", "tag", key)
```

I don't think logs need to be fragments? I asked about this under the PR with the logging RFD but I didn't get an answer. https://github.com/gravitational/teleport/pull/34173#issuecomment-2025348099

Reply:
I've been leaning towards capitalising the first letter in the conversions I've done - but - I'm not sure I have a particular preference beyond picking one and trying to run with it across the codebase.